C-Plus-Plus/cp/rain_water_trapping.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+    To find the total amount of water collected, we
+    firstly take an input of an array of size n of 
+    heights of buildings. After that we will try 
+    to find out minimum heights upto which water
+    could be stored at that particulat index of building.
+    This process has to be done from both left and the 
+    right sides. 
+    The complexity of the whole process will be O(n)
+    where, n stands for total number of buildings in each test-case.
+
+*/
+#include<bits/stdc++.h>
+using namespace std;
+
+// Function to return amount of water trapped
+void solve()
+{
+    long long int no_of_buildings;
+    //taking input of number of buildings
+    cin>>no_of_buildings;
+    // Making an array containing heights of buildings
+    long long int height[no_of_buildings];
+    for(long long int i=0;i<no_of_buildings;i++){
+        
+        //Taking input of height of ith-building
+        cin>>height[i];
+    }
```

Comment:
Use cout to display message how you are reading inputs

Reply:
I have done the required changes, please review.